pkg/devices.go

```diff
@@ -0,0 +1,210 @@
+package pkg
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	cdispec "github.com/container-orchestrated-devices/container-device-interface/specs-go"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	root = "/etc/cdi"
```

Comment:
We can have a `New()`  function that takes this as an option while defaulting to this directory if not set.